cdap-docs/cdap-apps/source/tracker/index.rst

```diff
@@ -0,0 +1,441 @@
+  systems such as `Cloudera Navigator <https://www.cloudera.com/products/cloudera-navigator.html>`__ 
+  for centralizing metadata repository and the delivery of complete, accurate, and correct
+  data.
+
+
+Example Use Case
+----------------
+An example use case describes how Tracker was employed in the `data cleansing and validating of
+three billion records <http://customers.cask.co/rs/882-OYR-915/images/tracker-casestudy1.pdf>`__.
+
+
+Tracker Application
+===================
+The Cask Tracker application consists of an application in CDAP with two programs and two datasets:
+
+- ``_Tracker`` application: names begins with an underscore
+- ``AuditLog`` service: it exposes the Tracker audit log as an API
+- ``AuditLogFlow`` flow: subscribes to Kafka audit messages and stores them in the ``AuditLog``	dataset
+- ``AuditLog`` custom dataset: type ``co.cask.tracker.entity.AuditLogTable``
+- ``kafkaOffset`` dataset: type key-value table
+
+The Tracker UI is shipped with CDAP, started automatically as part of the CDAP UI, and is available at:
+
+  http://localhost:9999/ns/default/tracker/home
+  
+or (Distributed CDAP):
+
+  http://host:dashboard-bind-port/ns/default/tracker/home
+  
+
+Tracker is built from a system artifact included with CDAP, |literal-cask-tracker-version-jar|.
+
+
+Installation
+============
+Cask Tracker is deployed from its system artifact included with CDAP. A CDAP administrator
+does not need to build anything to add Cask Tracker to CDAP; they merely need to add two
+properties to the ``cdap-site.xml`` and the enable the the application after starting CDAP.
+
+These two properties are required in the ``cdap-site.xml`` for the audit log functionality of Tracker::
+  
+  <!-- Audit Configuration -->
+
+  <property>
+    <name>audit.enabled</name>
+    <value>true</value>
+    <description>
+      Determines whether to publish audit messages to Apache Kafka
+    </description>
+  </property>
+
+  <property>
+    <name>audit.kafka.topic</name>
+    <value>audit</value>
+    <description>
+      Apache Kafka topic name to which audit messages are published
+    </description>
+  </property>
+
+Enabling Tracker
+----------------
+To enable Tracker, go to the **Tracker UI** at 
+(in the SDK) http://localhost:9999/ns/default/tracker/home or 
+(Distributed CDAP) http://host:dashboard-bind-port/ns/default/tracker/home 
+and press the ``"Enable Tracker"`` button to deploy and start Tracker.
+
+Once pressed, the application will be deployed, the datasets created (if necessary), the flow and service
+started, and search and audit logging will become available.
+
+If you are enabling Tracker from outside the UI, you will, in addition to enabling auditing 
+in the ``cdap-site.xml`` as described above, need to:
+
+- load the artifact (|literal-cask-tracker-version-jar|)
+  
+- create the application based on the artifact, supplying a configuration file::
+    
+    {
+      "config": {
```

Comment:
there are more config options including starting it with a kafka broker string. should we link to the tracker readme?